exercise_solutions.tex

```diff
@@ -1441,6 +1441,245 @@ \subsection*{Solution to \cref{ex:acconn}}
+that is $x \defeq j_n(y)$.
+Then we note that using the commutative diagram given by $\glue_n(x)$:
+\begin{center}
+\begin{tikzpicture}
+\node (N0) at (0,2) {$\Sn^{n}$};
+\node (N1) at (2,2) {$\Sn^{n+1}$};
+\node (N2) at (2,0) {$\Sn^{\infty}$};
+\node (N3) at (1.1,1.1) {};
+\node (N4) at (1.5,1.5) {};
+\draw[->] (N0) -- node[above]{\footnotesize $i_n$} (N1);
+\draw[->] (N0) -- node[left]{\footnotesize $j_n$} (N2);
+\draw[->] (N1) -- node[right]{\footnotesize $j_{n+1}$} (N2);
+\draw[double, double equal sign distance] (N3) -- node[left,above]{\footnotesize $glue_n$} (N4);
+\end{tikzpicture}
+\end{center}
+we get a path $x \defeq j_n(y) = j_{n+1} \circ i_n(y)$.
+But, as we will show, the image of $\Sn^n$ under $i_n$ is contractible,
+every point in the image is equal to $\north_{n+1}$. Thus, we are able to show that
+$j_{n+1}\circ i_n(y) = j_{n+1}(\north_{n+1})$. Composing the proof of the first step
+with the proof of the second step we conclude the exercise.
+
+To construct a $D_{\blank} : \prd{n:\nat} j_n(\north_n) = j_0(\north_0)$ we
+proceed by induction on $n$. For the base case we can 
+use $\refl{j_0(\north_0)}$. For the inductive case, we have by inductive
+hypothesis $j_n(\north_n) = j_0(\north_0)$. By our definition of $i_{\blank}$,
+we have that $\north_{n+1} \equiv i_n(\north_n)$. So $j_{n+1} (\north_{n+1})$
+equals $j_{n+1}\circ i_n(\north_n)$. By concatenation with $\glue_n(\north_n)$,
+we reduce our goal to the inductive hypothesis.
+
+Lets now show that the inclusion of $\Sn^n$ in $\Sn^{n+1}$ can be
+continuously retracted to $\north_{n+1}$.
+That is, lets construct a homotopy:
+\[ H_{\blank} : \prd{n:\nat}\prd{x:\Sn^n} i_n(x) = \north_{n+1} \]
+For the case $n\equiv 0$ we know that $i_0(\btrue) \equiv \north_1$ and
+$i_0(\bfalse) \equiv \south_1$.  This is because we constructed the inclusion
+that way. So we can prove the equalities
+using $\refl{\north_1}$ and $\merid_1(\btrue) : \north_1 = \south_1$.
+
+For the inductive case we defined, previously, $i_n(\north_n) \equiv \north_{n+1}$
+and $i_n(\south_{n}) \equiv \south_{n+1}$. So we can prove the equalities using
+$\refl{\north_{n+1}}$ and $(\merid_{n+1}(\north_n))^{-1}$.
+Then we have to prove that the function respects $\merid_n$:
+\[ \prd{x:\Sn^{n-1}} \dpath {x\mapsto (i_n(x) = \north_{n+1})} {\merid_n(x)} {\refl{\north_{n+1}}} {(\merid_{n+1}(\north_n))^{-1}} \]
+By \cref{thm:transport-path} (and some straightforward computation) this reduces to:
+\[ i_n(\merid_n(x)) = \merid_{n+1}(\north_{n}) \]
+But, by our definition of $i_{\blank}$, and the computation rule of the
+suspension induction $i_n(\merid_n(x))$ equals $\merid_{n+1} \circ i_{n-1}(x)$.
+And, by inductive hypothesis, $i_{n-1}(x) = \north_n$, which gives us the
+desired result.
+
+Composing the two proofs we just gave we get a function:
+\[ J_{\blank} \defeq \glue_n(x)\ct \apfunc{j_{n+1}}{H_n(x)}\ct D_{n+1}
+    : \prd{n:\nat}\prd{x:\Sn^{\infty}} j_n(x) = j_0(\north_0). \]
```

Comment:
Also, just a comment (not mandatory to change): the macro `\prd` is smart so that you don't need to write it twice: you can just say `\prd{n:\nat}{x:\Sn^n}`.